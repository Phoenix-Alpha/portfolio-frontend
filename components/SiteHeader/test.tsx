import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { SiteHeader } from './index'

describe('<SiteHeader />', () => {
  it('should renders correct <SiteHeader />', () => {
    const wrapper = shallow(<SiteHeader />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
