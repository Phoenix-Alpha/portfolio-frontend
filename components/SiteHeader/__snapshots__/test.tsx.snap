// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SiteHeader /> should renders correct <SiteHeader /> 1`] = `
<div
  className="sticky top-0 z-[9999]"
  style={
    Object {
      "background": "linear-gradient(99.72deg, #0D1641 0.86%, #0A1134 98.01%)",
    }
  }
>
  <header
    className="lg:flex lg:justify-between lg:items-center lg:mx-auto lg:flex-no-wrap"
  >
    <Container
      className="lg:flex lg:justify-between lg:items-center !px-5 lg:px-[50px] my-[18px] lg:my-8 mx-0 lg:mx-auto lg:flex-no-wrap"
    >
      <div
        className="flex lg:block justify-between items-center flex-no-wrap"
      >
        <div
          className="flex items-center w-full"
        >
          <SiteLogo />
          <div
            className="flex lg:hidden items-center mr-0 ml-auto"
          >
            <MenuToggleButton
              onButtonClick={[Function]}
            />
          </div>
        </div>
      </div>
      <MainNav
        activeItem={null}
        isShown={false}
        setIsShown={[Function]}
        toggleMainNavItem={[Function]}
      />
    </Container>
  </header>
</div>
`;
